test/test_fluid.py

```diff
@@ -76,11 +76,8 @@ def test_velocity_gradient_sanity(actx_factory, dim, mass_exp, vel_fac):
     velocity = vel_fac * nodes
     mom = mass * velocity
 
-    q = join_conserved(dim, mass=mass, energy=energy, momentum=mom)
-    cv = split_conserved(dim, q)
-
-    grad_q = obj_array_vectorize(discr.grad, q)
-    grad_cv = split_conserved(dim, grad_q)
+    cv = make_conserved(dim, mass=mass, energy=energy, momentum=mom)
+    grad_cv = make_conserved(dim, q=obj_array_vectorize(discr.grad, cv.join()))
```

Comment:
Is it ok to just use `discr.grad` here, with no flux terms?

Reply:
It is safe here where I am just testing the form and function of the product rule grad functions.  We just want the grad of a continuous function in space for that test.